sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -506,6 +511,20 @@ class EthService(
   @RpcMethod("net_version")
   def version: String = String.valueOf(networkParams.chainId)
 
+  private def getPeerCount: Int = {
+    val peerReq = (networkControllerRef ? GetConnectedPeers).asInstanceOf[Future[Seq[ConnectedPeer]]]
+    Await.result(peerReq, nvtimeout).size
+  }
+
+  @RpcMethod("net_peerCount")
+  def peerCount: Quantity = new Quantity(getPeerCount)
```

Comment:
After my RPC cleanup has been merged, `Quantity` does not exist anymore. Just return `BigInteger`, hex conversion happens automagically.